src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The link references a SCSS file (`main.scss`). Ensure that the SCSS is compiled to CSS and link the resulting CSS file instead. This is necessary because browsers do not understand SCSS directly.